src/eradius_proxy.erl

```diff
@@ -3,12 +3,18 @@
 -behaviour(eradius_server).
 -export([radius_request/3, validate_arguments/1]).
 
+-ifdef(TEST).
+-export([resolve_routes/4, validate_options/1, new_request/3,
+         get_key/4, strip/4]).
+-endif.
+
 -include("eradius_lib.hrl").
 -include("dictionary.hrl").
 
 -define(DEFAULT_TYPE, realm).
 -define(DEFAULT_STRIP, false).
 -define(DEFAULT_SEPARATOR, "@").
+
```

Comment:
is there a reason to have the default type, strip and separator as defines? I can see only one place where they are used.